### PR TITLE
Fix IE8 ES6 compilation errors.

### LINKED ISSUE #46: IE8 Object.defineProperty Error in utils/error-handler.js
In the compiled vendor.js file, the compiled js, ends up calling `Object.defineProperty(target, descriptor.key, descriptor);` and this throws an error in IE8.

This code does not seem to be in the actual error-handler file, I think it has something to do with the `class` definition. 

The full output is in this [gist](https://gist.github.com/xcskier56/1221f2cb1ffb727f59fc) on line 7 near the very end of the row.

### PR DESCRIPTION
**Pull request by xcskier56**
This fixes #46. By changing addon/utils/error-handler.js from a class to an ember
object, I have gotten around the compilation that output
Object.defineProperty() which breaks in IE8.

Additionally, I've changed the import/export statements to a slightly more verbose syntax, but one that will be less likely to have output that is incompatible with ie8.

I have tested this and confirmed that it fixes the issues that I was having with IE8.

### CODE REVIEW

File: addon/services/trackjs.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Ember from 'ember';
  * enough grief in trying to get this to work that this proxy seemed like the
  * easiest solution for now.
  */
-export default Ember.Service.extend({
+var TrackJS = Ember.Service.extend({
```
Comment (jherdman) [inline addon/services/trackjs.js]: Why this change?

File: addon/utils/error-handler.js [hunk 2]
```diff
@@ -2,18 +2,18 @@ import Ember from 'ember';
 
 const Logger = Ember.Logger;
 
-class ErrorHandler {
-  constructor(reporter) {
-    this.reporter = reporter;
-  }
+var ErrorHandler = Ember.Object.extend({
```
Comment (jherdman) [inline addon/utils/error-handler.js]: var => const

File: addon/utils/error-handler.js [hunk 3]
```diff
@@ -2,18 +2,18 @@ import Ember from 'ember';
...
+  init: function() {
+    this.reporter = this.get('reporter');
+  },
 
-  report(error) {
+  report: function(error) {
```
Comment (jherdman) [inline addon/utils/error-handler.js]: Revert.

File: app/utils/error-handler.js [hunk 4]
```diff
@@ -1 +1,3 @@
-export { default } from 'ember-cli-trackjs/utils/error-handler';
+import ErrorHandler from 'ember-cli-trackjs/utils/error-handler';
+
+export default ErrorHandler;
```
Comment (jherdman) [inline app/utils/error-handler.js]: What's the motivation behind this change?

**Comment by jherdman** [comment]
I'm considering closing this pull request as MS is shuttering support for IE8. Is this change still needed for you?

**Comment by xcskier56** [comment]
I'm not sure why, but I seem to not be getting errors from this anymore. While we still have to support IE8, I don't see any need for this PR right now. 

If in the future I start having issues with this I will revisit. Thanks for the work on this!
